eox_tagging/models.py

```diff
@@ -37,7 +37,7 @@ class TagQuerySet(QuerySet):
     def create_tag(self, **kwargs):
         """Method used to create tags."""
         target = kwargs.pop("target_object", None)
-        if target and target.__class__.__name__ in OPAQUE_KEY_PROXY_MODEL_TARGETS:
+        if target and target.__class__.__name__.lower() in OPAQUE_KEY_PROXY_MODEL_TARGETS:
```

Comment:
```suggestion
        if type(target).__name__.lower() in OPAQUE_KEY_PROXY_MODEL_TARGETS:
```
If we are only working with python3.

Reply:
Maybe this is not the PR to make a breakable change :sweat_smile:

Reply:
I think the use of swagger already made us drop python 2.7 support so it shouldn't break anything. But is true that `.__class__.__name__` and other similar patterns happen in several places so it should be handled in another PR.